.pre-commit-config.yaml

```diff
@@ -43,3 +48,8 @@ repos:
     hooks:
     -   id: mypy
         exclude: tests/
+-   repo: https://github.com/PrincetonUniversity/blocklint
+    rev: v0.2.5
+    hooks:
+    -   id: blocklint
+        exclude: 'docs/Makefile|CHANGELOG.rst|tox.ini|41.internal.rst'
```

Comment:
Is that `41.internal.rst` needed?

Reply:
That's for `blocklint`. I figured I'll be coming back through adding the normal `release_notes` docs process here in a bit.